include/Model/Languages.php

```diff
@@ -321,7 +321,7 @@ public function update( $args ) {
 			// Update domains.
 			$domains = $this->options->get( 'domains' );
 
-			if ( ! empty( $domains[ $old_slug ] ) ) {
+			if ( isset( $domains[ $old_slug ] ) ) {
```

Comment:
Should we use `is_numeric()` or `is_int()`?

Reply:
No need IMHO.
1. If the options have not been saved yet since the creation of this language, `$domains[ $old_slug ]` is not set, which means we must keep `isset()`.
2. In the other cases, the value of `$domains[ $old_slug ]` comes from options, which are already validated and sanitized, so we know it is an integer: no need of an extra `is_int()` then.

Reply:
I see, sorry I messed up with menu.
I have to say I don't understand the link between what you're fixing and menu issue you're showing in the PR description.

Reply:
Probably because I'm the one who messed up, not you.

Reply:
🤣